README.md

```diff
@@ -5,9 +5,48 @@
 
 ## Overview
 
-The python package `libpyvinyl` exposes the high level API for simulation codes under
-the umbrella of the Virtual Neutron and x-raY Laboratory (ViNYL).
+Simulations provide invaluable insights to plan (before) and
+ understand (after) experiments at neutron and X-ray facilities. 
+A wide set of libraries and programs are already available to simulate
+ neutron and X-ray beams, propagate them through the beamlines,
+ interact with matter of a sample and get data acquired by detectors. 
+
+The aim of this library is to provide a high level API to allow users
+ to access simulations with a unified interface, reaching almost
+ seamless interoperability, allowing to chain multiple
+ simulation softwares in a natural way. 
```

Comment:
The aim of  *libpyvinyl* is to provide a high level neutron and x-ray simulation API. With this harmonized user interface we achieve seamless interoperability of individual simulations thereby facilitating the concatenation of simulation steps into a simulation pipeline.

Reply:
ok